src/dashboard/Settings/AppleCerts.react.js

```diff
@@ -63,8 +63,8 @@ export default class AppleCerts extends React.Component {
             type={Modal.Types.DANGER}
             title='Delete this certificate'
             subtitle='Notifications will no longer be sent to the associated app.'
-            cancelText={'Cancel'}
-            confirmText={'Delete it!'}
+            cancelText='Cancel'
+            confirmText='Yes'
```

Comment:
I read somewhere in the Apple guidelines that the default? (or all? not remember) button label should be the action like "Delete, Submit, Copy, Clone, etc.. " But I am not sure and I am lazy to dig into the Microsoft/Apple UII guidelines and I don't feel we need to go so fine-tuning right now, so I am very happy with what you propose above.
Fixed!

Reply:
I think that makes sense because it emphasizes the action on the button in a single word. If you don't mind, do you want to change the buttons from "yes" to the corresponding action? If not, we never mind, we can merge the PR as it is already an improvement.

Reply:
Done!